Museum

Marlyn-Mayienga opened the pull request:
# Historia Program Website 
In this module, we did:
- created the home page and modified the mobile view
- created the about page and modified the mobile view
- modified the readme file to define what the website is about
- deployed the project to GitHub pages

Marlyn-Mayienga committed:
made changes to the mobile view of the home and about pages

Marlyn-Mayienga committed:
modified the readme file

Marlyn-Mayienga committed:
renamed home.html to inex.html

Marlyn-Mayienga committed:
modified the close icon for the navbar

Marlyn-Mayienga committed:
fixed the linter errors

Marlyn-Mayienga committed:
made the following changes:
modified the speakers-feature
modified the home page
modified the readme file
added thw js linters

Marlyn-Mayienga committed:
added the loom recording link

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,269 @@
+        </div>
+      </div>
+      <div class="container-fluid second-nav">
+        <a class="navbar-brand hide-sm" href="home.html">
```
- [ ] I see that you are trying to add the link to return to the homepage. The link is broken because you are referencing `home.html` instead of  `index.html`. You don't have `home.html` in your project so the link doesn't work. Kindly fix it 😄

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,269 @@
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon">
+            <i class="menuIcon fas fa-bars"></i>
```
- [ ] Kindly remove the white background on the hamburger, and also make the hamburger color a bit transparent or the same color/pattern as the top section background.  Please let the hamburger be on the page background so that it matches the design template 🤓 

|  Design Template   | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181388913-d79328b9-e48a-48dc-b3a4-9bc2264c4bad.png)  | ![image](https://user-images.githubusercontent.com/63473584/181388856-77b5f763-a460-4633-98b4-0efaeac5ee47.png)  |

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,312 @@
+        </div>
+      </section>
+      <section class="program text-center">
+        <p class="program-title">Main Program</p>
```
- [ ] Kindly increase the font size of the **Main Program** heading such that it matches the design template as It is currently too small. It will also be better if you use the heading tag (h2 -h6) for the section header instead of the  `p tag` to make your website semantic 👍

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,312 @@
+          <hr />
+        </div>
+        <div class="row justify-content-center">
+          <div class="col-md-2 px-0">
+            <div class="card schedule-card">
+              <img
+                src="./assets/program_icon_01.png"
+                class="card-img-top"
+                alt="icon"
+              />
+              <div class="card-body d-flex flex-column align-items-center">
+                <h5 class="card-title">Lecture</h5>
+                <p class="card-text">
+                  Learn about current exhibitions, conservation projects, the
+                  Museum's rich collection, and other topics through talks by
+                  leading experts.
+                </p>
+              </div>
+            </div>
+          </div>
```
- [ ] Please ensure that each program articles have its elements organized in a horizontal row, with the image at the left, title in the center, and description at the right 😉 .

|  Design Template   | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181392057-d0a700f5-4127-445c-a458-4ecf3e65f2f1.png)  | ![image](https://user-images.githubusercontent.com/63473584/181392216-3bdb567f-ab1d-42e2-96bb-4769ca6599db.png)  |

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,312 @@
+      </section>
+      <section class="guests text-center">
+        <div class="container">
+          <h2 class="guests-title">Featured speakers</h2>
```
- [ ] Kindly add some margins to the featured speakers so that it won't be too close to the other sections as it is on the design template

|  Design Template   | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/181393011-1af7c080-ec37-42ce-a004-56e12c3a953a.png)  | ![image](https://user-images.githubusercontent.com/63473584/181393076-376dd617-ad0d-4aca-badc-479f2a3bdc6a.png)  |

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,269 @@
+    <!-- WEBSITE CONTENT  -->
+
+    <div class="container-fluid main-content px-0">
+      <section class="introduction">
```
- [ ] Kindly ensure that your introduction section is well centered in the top section of the page. It is currently more aligned to the right. Kindly fix it 👇 

![image](https://user-images.githubusercontent.com/63473584/181393425-b409a7bc-d5cb-499b-8792-4d9e057e9843.png)

victorokoroji commented on index.html:
```diff
@@ -0,0 +1,312 @@
+          <div class="row justify-content-center">
+            <div class="col-md-9">
+              <h1 class="history intro-greeting">"Hello! Sharing World"</h1>
+              <h4 class="history intro-tag">
+                HISTORIA MAMBOLEO ART AND CULTURAL SUMMIT 2022
+              </h4>
```
- [ ] To make your website look more attractive, I would recommend you use **cocogoose** font for the title as well as the summit date 😄

victorokoroji commented on styles.css:
```diff
@@ -0,0 +1,378 @@
+  text-align: left;
+}
+
+@media only screen and (max-width: 768px) {
```
- [ ] It would be nice if your styling follows the **mobile-first** approach which means that your media queries should change the design from mobile to desktop, not the other way around. Kindy consider that 👍

victorokoroji reviewed (changes requested):
Hi @Marlyn-Mayienga , 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- Design is good ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅  
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015),  as required by the [Creative Commons License](https://creativecommons.org/licenses/) in the acknowledgment section of your readme 👍 .

_Check the comments under the review._
<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Marlyn-Mayienga committed:
made the following changes:
fixed the mobile-first approach
fixed the program schedule
fixed the font for the headline
fixed the navbar whitespace

Marlyn-Mayienga committed:
modified the acknowledgement of the readme file

Marlyn-Mayienga committed:
fix css linters

Marlyn-Mayienga committed:
fix linters

ShahierNashaat reviewed (approved):
Hi @Marlyn-Mayienga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
